types/setting.go

```diff
@@ -73,6 +73,7 @@ const (
 	SettingNameBackingImageRecoveryWaitInterval             = SettingName("backing-image-recovery-wait-interval")
 	SettingNameGuaranteedEngineManagerCPU                   = SettingName("guaranteed-engine-manager-cpu")
 	SettingNameGuaranteedReplicaManagerCPU                  = SettingName("guaranteed-replica-manager-cpu")
+	SettingNameOrphanAutoDeletion                           = SettingName("orphan-auto-deletion")
```

Comment:
Need to add this setting CR in `deploy/install/01-prerequisite/04-default-setting.yaml`.